integreat_cms/core/settings.py

```diff
@@ -144,6 +144,16 @@
 #: (see :mod:`~integreat_cms.gvz_api` for more information).
 GVZ_API_URL = "https://gvz.integreat-app.de"
 
+##############
+# Matomo API #
+##############
+
+#: Whether or not the Matomo API is enabled.
+#: This is used for statistics evaluations of the API hits
+MATOMO_API_ENABLED = bool(
+    strtobool(os.environ.get("INTEGREAT_CMS_MATOMO_API_ENABLED", "True"))
+)
+
```

Comment:
```suggestion
```
I thought you wanted to remove the option instead of making use of it? (see https://github.com/digitalfabrik/integreat-cms/pull/1758#discussion_r1019034863)